addOns/selenium/src/main/java/org/zaproxy/zap/extension/selenium/Browser.java

```diff
@@ -224,7 +212,6 @@ public static String getBundledWebDriverPath(Browser browser) {
                 logger.warn("Failed to set the bundled WebDriver executable:", e);
             }
         }
-
```

Comment:
Revert.

Reply:
Not sure I understand, are you asking me to revert the whole file?